07_deploy_masters.sh

```diff
@@ -67,6 +67,11 @@ done
 
 wait_for_bootstrap_event
 
+# Image registry should temporarily use emptyDirs
```

Comment:
Is it worth mentioning here that eventually rook will be deployed first so this won't be needed?

Reply:
Reworked this to use assets, added a comment in the template